app/controllers/reservations_controller.rb

```diff
@@ -0,0 +1,21 @@
+class ReservationsController < ApplicationController
+  def create
+    reservation = Reservation.new(reservation_params)
+    reservation.confirmed!
+    nights = Night.where(couch_id: night_params["couch_id"]).where(date: Date.parse(params["check_in"])..Date.parse(params["check_out"]) - 1.day)
+    reservation.nights << nights
+    reservation.save
+    flash[:success] = "#{Couch.find(params["couch_id"]).name} reserved for #{params["check_in"]}."
```

Comment:
should we have sad path (if/else) in case the reservation doesn't save successfully?